site/public/css/navigation.css

```diff
@@ -62,21 +71,82 @@
     display: none;
 }
 
-@media (min-width: 541px) {   
+.course-section-title {
+    display: inline;
+}
+
+@media (max-width: 401px) {
     .course-section-heading {
-        border-bottom: 2px solid #000;
-        background: none;
+        font-size: 14px;
     }
+    .course-main .course-title {
+        font-size: 12px;
+    }
+    .course-button .btn, .subtitle {
+        font-size: 11px;
+    }
+}
 
-    .gradeable-row {
-        border: none;
+@media (max-width: 950px) {
+    .course-section-heading {
+        margin: 10px -25px;
+        font-weight: 400;
+        background-color: var(--default-white);
+        transition: 0.2s;
+        border: 1px solid var(--text-black);
     }
 
-    .btn-nav {
-        min-width: 95%;
+    .course-section-heading:hover {
+        box-shadow: 1px 2px 3px var(--text-black);
+        background: var(--submitty-logo-blue);
+        color: white;
+        translate: 0 -3px;
+    }
+
+    .course-section {
+       display: none;
+    }
+
+    .panel-head-active {
+        box-shadow: 1px 2px 3px var(--text-black);
+        background: var(--submitty-logo-blue);
+        color: white;
+    }
+
+    .panel-head-active .expand-icon, .collapse-icon {
+        display: inline;
+    }
+
+    .panel-head-active .collapse-icon, .expand-icon {
+        display: none;
+    }
+
+    .panel-active {
+        display: block;
+    }
+
+    #gradeables-content h1 {
+        font-weight: 400;
+    }
+
+    .course-main .course-title {
+        font-size: 14px;
+    }
+    .course-button .btn, .subtitle {
+        font-size: 13px;
     }
 }
 
+/*@media (min-width: 780px) {*/
+/*    .gradeable-row {*/
+/*        border: none;*/
+/*    }*/
+
+/*    .btn-nav {*/
+/*        min-width: 95%;*/
+/*    }*/
+/*}*/
+
```

Comment:
You should just remove this extra commented out css, unless there is a good reason for it to stay here that I don't know about.

Reply:
It's not needed, removed.